### PR TITLE
Dir refactor

### PR DESCRIPTION
**Pull request by hopkiw**
Layout changes prior to moving to new repository. Allows performing builds and tests in a simple debian container.

### CODE REVIEW

**Comment by zmarano** [comment]
Update the README to note changed directories (Overview section, Utilities Directories, ...)

**Comment by hopkiw** [comment]
> Update the README to note changed directories (Overview section, Utilities Directories, ...)

Done

File: packages/google-compute-engine-oslogin/README.md [hunk 1]
```diff
@@ -1,76 +1,73 @@
...
+`.rpm` packages for the OS Login components.
 
 ## Components
 
 #### Authorized Keys Command
 
 The `google_authorized_keys` binary is designed to be used with the sshd
-[AuthorizedKeysCommand](https://linux.die.net/man/5/sshd_config) option in
-`sshd_config`. It does the following:
+`AuthorizedKeysCommand` option in [sshd_config(5)](https://linux.die.net/man/5/sshd_config)
```
Comment (illfelder) [inline packages/google-compute-engine-oslogin/README.md]: Add a period at the end of line 51.

File: packages/google-compute-engine-oslogin/README.md [hunk 2]
```diff
@@ -1,76 +1,73 @@
...
     ```
     http://metadata.google.internal/computeMetadata/v1/oslogin/users?username=<username>
     ```
-*   Checks to make sure that the user is authorized to log in.
+*   Checks to make sure that the user is authorized to log in:
     ```
     http://metadata.google.internal/computeMetadata/v1/oslogin/authorize?email=<user_email>&policy=login
     ```
 *   If the check is successful, returns the SSH keys associated with the user
     for use by sshd.
```
Comment (illfelder) [inline packages/google-compute-engine-oslogin/README.md]: Might be useful to note that the authorized keys command will return a non-zero exit code if the authorization check fails.

File: packages/google-compute-engine-oslogin/README.md [hunk 3]
```diff
@@ -1,76 +1,73 @@
...
 
-The `nss_oslogin` module is built and installed in the appropriate `lib`
-directory as a shared object with the name `libnss_oslogin.so.2`. The module is
-then activated by an `oslogin` entry in `/etc/nsswitch.conf`. The NSS module
-supports looking up `passwd` entries from the metadata server via
-`getent passwd`.
+`libnss_oslogin.so` and `libnss_cache_oslogin.so` are NSS service modules which
+make OS Login users and groups available for use on the local system. The module
+is activated by adding `oslogin` and `cache_oslogin` entries for services in
+[nsswitch.conf(5)](https://linux.die.net/man/5/nsswitch.conf).
```
Comment (illfelder) [inline packages/google-compute-engine-oslogin/README.md]: Please add a closing `>` to line 78.

**Pull request merged**